practices/images-best-practices.md

```diff
@@ -0,0 +1,48 @@
+# Best practices when dealing with images for the Web
+
+In a front-end workflow we typically deal with two kinds of images:
+
+* **Icons, logos, buttons and other graphical elements** that are part of the User Interface of the page. These are usually managed by the frontend, and so they are stored with the rest of resources (templates, CSS, JS). As many of those resources, they may be shared between many different pages.
+
+* **Content images**. These are unique to a page (or a handful of pages). They will be usually provided from a database or a content management system.
+
+Most best practices are common to both categories.
+
+## Formats
+
+### What format to use
+
+* Use SVGs whenever you want to display a vector image.
```

Comment:
Hmm... I see what you mean. When I wrote this I was concerned about the source of the image though, as that usually is not under our control: we can only use SVG if we have the original image in a vector format (AI or whatever) as there's no (easy) way back from a bitmap image to a vector one.

So, at risk of being redundant, what I wanted to say is "if you're lucky enough to have a vector image and the original vector format (AI or whatever) then just use that one. Don't convert it to a bitmap-based format."